### ISSUE TITLE
error for forecasting-with-panel-datasets example 

### ISSUE THREAD

**Issue by Sandy4321** [microsoft/FLAML#1161]
I got error for forecasting-with-panel-datasets example 
https://microsoft.github.io/FLAML/docs/Examples/AutoML-Time%20series%20forecast#forecasting-with-panel-datasets

I have windows 11
FLAML   1.2.4

![image](https://github.com/microsoft/FLAML/assets/11426119/a8c2992e-feff-4052-b046-a91a130ed103)
builtins.FileNotFoundError: Could not find module 'C:\my_py_environments\py310_env_apr2023\Lib\site-packages\torchaudio\lib\libtorchaudio.pyd' (or one of its dependencies). Try using the full path with constructor syntax.

**Comment by Sandy4321** [comment]
error in line 
data, special_days = get_stalliion_data()
![image](https://github.com/microsoft/FLAML/assets/11426119/a3b25f91-a5d9-4573-84ad-1fb96c41f9e6)

**Comment by Sandy4321** [comment]
ok 
i created new enviroment only for flaml and this error gone
but now there is another error
builtins.TypeError: Trainer.__init__() got an unexpected keyword argument 'gpus'

![image](https://github.com/microsoft/FLAML/assets/11426119/919930fe-a77b-4e82-94e7-c562e9df4270)

**Comment by sun-qisen** [comment]
I have encountered the same problem, in fact, it is caused by the pytorch-lightning version, which can be solved by reducing the version, but I still encounter a new environmental problem. Finally through the develop history I find the environment requirements, you can just run:
`pip install pytorch-forecasting==0.9.0 pytorch-lightning==1.5.10`

**Comment by int-chaos** [comment]
yes thank you for the comment, we are experiencing issues with the new pytorch-lightning version (as discussed in the discord group)
